pkg/mbedtls/contrib/entropy_mbedtls_riot.c

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
Shouldn't this be its on module so in the future people can use `mbedtls` without the entropy source (if they so desire)?

So `pkg/mbedtls/contrib/mbedtls_entropy_source/...`. This way you also might not need to define pseudo-modules.

Again: see [lwIP](https://github.com/RIOT-OS/RIOT/blob/b45363b7844b6375239980d62d76811ae30b52a9/pkg/lwip/Makefile.include#L29-L37) how to implement actual modules within the `contrib` directory of a package.

Reply:
Good point. I have tried this in my last commit, mind to have another look?